doc/specific_iocs/motors/Motors-Trouble-Shooting.md

```diff
@@ -96,12 +96,6 @@ in the logs. To fix this you must ask electronics to confirm the jumper location
 
 There is a Galil specific PV called `MTRXXXX_AUTOONOFF_CMD` which controls whether an axis automatically powers up when given a move (found at the bottom of the motor details OPI). This should normally be `On` so that the motor energises before a move and switches off after a move. In some cases, the motor axis needs to be energised for the whole time, when the axis is under great load or the motor is old. In this case, the auto de-energise should be `No` the energised state should be `energise`. The "motor off deadband" should also be set to a negative number, -1, so that the motor does not turn itself off after a move, this is found in the engineering screen, `Motor off deadband`. If you don't set this to -ve it may not deenergise the motor but this will be unreliable, we think when you don't have an encoder it won't deenergise the motor. There is a warning that reads `Motor off deadband is not small enough, motor may deenergise after move.` which will show in this later case.
 
-### The axis will not move, a message gets put in the log of "move begin failure axis X after XXX seconds: ... [Decelerating or stopped by FWD limit switch or soft limit FL]"
-
```

Comment:
I think it would be useful to keep a small section here with this heading, it could still be a hard limit or even a soft limit has still bee hit, it is just that this should now be a user specified soft limit and not a phantom one caused by a drift so all the sync tolerance stuff can be replaced with a reference to the other section added above.

Reply:
Done - but moved to galil page rather than motors page as this error is galil-specific.